projects/developer/src/app/navbar/navbar.component.scss

```diff
@@ -8,6 +8,105 @@ $navbar-item-width: 150px;
 
 .navbar-ctr {
     height: $navbar-height;
+    position: relative;
+}
+
+.navbar-ctr--with-banner {
+    margin-top: $navbar-height;
+}
+
+.navbar-banner {

```

Comment:
nothing to resolve here, but just so you are aware in the future, we have primeflex installed - just some flexbox helper classes https://github.com/primefaces/primeflex

Reply:
Thank you!  I'll read up on it.